API.md

```diff
@@ -65,13 +65,13 @@ After running the previous command, you should see output similar to the followi
 ```
 *   Trying [::1]:9000...
 * Connected to localhost (::1) port 9000
-> GET /irods-http-api/0.1.0/authenticate HTTP/1.1
+> GET /irods-http-api/0.2.0/authenticate HTTP/1.1
 > Host: localhost:9000
 > User-Agent: curl/8.4.0
 > Accept: */*
 > 
 < HTTP/1.1 302 Found
-< Server: irods_http_api/0.1.0 [d6164b7aa421265e0f9f9d5f7eb69bb67d8e4a3c]
+< Server: irods_http_api/0.2.0 [d6164b7aa421265e0f9f9d5f7eb69bb67d8e4a3c]
```

Comment:
Oh. That SHA is not in sync. Consider replacing it with `<build_sha>`. The version number could be replaced with `<version>` so we don't have to worry about it anymore too.

Reply:
Forgot to mention the SHA should also be in parens rather than square brackets. That was changed a few weeks ago.

Reply:
oh, I see - just punt on keeping it 'true'.